From 640cb10d0ef4d375c98a4459716f2df78cd652b9 Mon Sep 17 00:00:00 2001
From: David Milburn <dmilburn@redhat.com>
Date: Mon, 18 Aug 2014 19:58:47 -0400
Subject: [block] nvme: Delete NVME_GET_FEAT_TEMP_THRESH

Message-id: <1408391935-24886-15-git-send-email-dmilburn@redhat.com>
Patchwork-id: 87887
O-Subject: [RHEL7.1 PATCH BZ 1111259 14/22] NVMe: Delete NVME_GET_FEAT_TEMP_THRESH
Bugzilla: 1111259
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Matthew Wilcox <matthew.r.wilcox@intel.com>

This define isn't used, and any code that wanted to use it should use
NVME_FEAT_TEMP_THRESH instead.

Signed-off-by: Matthew Wilcox <matthew.r.wilcox@intel.com>
(cherry picked from commit de672b9748f78dcbc663e12ea44cb24dc287baf0)
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/block/nvme-scsi.c b/drivers/block/nvme-scsi.c
index 7ec61c4..13403ee 100644
--- a/drivers/block/nvme-scsi.c
+++ b/drivers/block/nvme-scsi.c
@@ -240,7 +240,6 @@ static int sg_version_num = 30534; /* 2 digits for each component */
 
 /* NVMe Namespace and Command Defines */
 #define NVME_GET_SMART_LOG_PAGE    0x02
-#define NVME_GET_FEAT_TEMP_THRESH   0x04
 #define BYTES_TO_DWORDS     4
 #define NVME_MAX_FIRMWARE_SLOT    7
 
-- 
1.7.1